src/main/java/com/google/sps/data/DBUtils.java

```diff
@@ -27,6 +37,8 @@ public static void productionMode() {
     database = FirestoreClient.getFirestore();
     recipesReference = database.collection("recipes");
     usersReference = database.collection("users");
+    recipeMetadataReference = database.collection("recipeMetadata");
+    tagsReference = database.collection("tags");
   }
 
   public static Firestore db() {
```

Comment:
This method seems to be unused as we set our database to public, so we can delete this method.